website-custom-views/src/content/development/permissions.mdx

```diff
@@ -23,38 +23,6 @@ The `PERMISSIONS` variable contains a `View` and `Manage` properties:
 
 You can then use the `PERMISSIONS` variable to reference the permission in the application code.
 
-<Info>
```

Comment:
@emmenko I removed this info from all places because the version `21.21.0` is specific to Custom Applications. 

We can keep the of the content, if you think it's still applicable.